utils/filesystem.go

```diff
@@ -104,3 +111,81 @@ func PingHealth(healthEndpoint string) bool {
 	}
 	return started
 }
+
+//GetZipURL from github api /repos/:owner/:repo/:archive_format/:ref
+func GetZipURL(c *cli.Context) string {
+	branch := c.String("branch")
+	owner := c.String("owner")
+	repo := c.String("repo")
+
+	client := github.NewClient(nil)
+
+	opt := &github.RepositoryContentGetOptions{Ref: branch}
+
+	URL, _, err := client.Repositories.GetArchiveLink(context.Background(), owner, repo, "zipball", opt)
+	if err != nil {
+		log.Fatal(err)
+	}
+	url := URL.String()
+	fmt.Println(URL)
+	return url
+}
+
+//DownloadFile - the git zip of master into current dir
```

Comment:
Similar to the above comment about this function comment.

Reply:
Can probably make this a bit clearer